source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html

```diff
@@ -290,7 +296,7 @@
             <td>21</td>
             <td>@ocramius</td>
             <td>Twitter</td>
-            <td>b1fc7171/ocramius-macbook</td>
+            <td>b1fc7171/ ocramius-macbook</td>
```

Comment:
These spaces are not supposed to be there (assuming session ID contains device ID)